optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/api/PlannerBenchmark.java

```diff
@@ -25,13 +27,17 @@
 
     /**
      * Run all the single benchmarks and create an overview report.
+     *
+     * @return never null, path to the directory in which the benchmark results are stored
      */
-    void benchmark();
+    Path benchmark();
```

Comment:
Why a Path and not a File? Because Path is the new thing.
But the entire optaplanner-benchmark API uses File currently.
Let's **not** mix them.

Proposal A) Return Path (mix them)
Proposal B) Return Path + refactor all usages of File on the benchmark api to use Path instead. This breaks BC. Not possible for 8.X
Proposal C) Return File for now, create an issue linked to PLANNER-2295 to refactor optaplanner-benchmark to use Path instead of File.

Reply:
I think we need to do C). Wdyt?